blobs/t440p/download-clean-me

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+set -e
+
+function usage() {
+    echo -n \
+        "Usage: $(basename "$0")
+Download Intel ME firmware from Lenovo, neutralize, and shrink.
+"
+}
+
+ME_BIN_HASH="b7cf4c0cf514bbf279d9fddb12c34fca5c1c23e94b000c26275369b924ab9c25"
+
+if [[ "${BASH_SOURCE[0]}" == "$0" ]]; then
+    if [[ "${1:-}" == "--help" ]]; then
+        usage
+    else
+        BLOB_DIR="$(cd "$(dirname "$0")" && pwd)"
+        COREBOOT_DIR="$(find "${BLOB_DIR}/../../build/x86/" -maxdepth 1 -type d -name 'coreboot-*')"
```

Comment:
@rbreslow Not the best approach for CircleCI usecase, where just like local test build VM, contains multiple coreboot versions. Better to have me_cleaner locally/and/or move xx30 one under blobs/ and reuse it adapting other scripts to use it instead (where xx20 is patched to deal with invalid ME images and fill what is missing)

To show the problem with actual script:
```
user@heads-tests:~/heads$ blobs/t440p/download-clean-me
/tmp/tmp.Jvah5aqgmg ~/heads
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 5262k  100 5262k    0     0  2991k      0  0:00:01  0:00:01 --:--:-- 2991k
Extracting "Intel Management Engine 9.1 Firmware for Windows 7/8.1/10" - setup data version 5.4.2
 - "app/FwDetect.exe"
 - "app/FwUpdate.exe"
 - "app/FWUpdLcl.exe"
 - "app/FWUpdLcl64.exe"
 - "app/Idrvdll.dll"
 - "app/ME9.1_5M_Production.bin"
 - "app/MEInfoWin.exe"
 - "app/MEUpdate.CMD"
 - "app/Pmxdll.dll"
 - "app/SLA_TOOLS.pdf"
Done.
mv: cannot move 'app/ME9.1_5M_Production.bin' to '/home/user/heads/blobs/t440p/../../build/x86/coreboot-4.11'$'\n''/home/user/heads/blobs/t440p/../../build/x86/coreboot-4.17'$'\n''/home/user/heads/blobs/t440p/../../build/x86/coreboot-4.18'$'\n''/home/user/heads/blobs/t440p/../../build/x86/coreboot-4.13/util/me_cleaner': No such file or directory

```